x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/custom_patterns_input/custom_patterns_input.js

```diff
@@ -33,11 +33,13 @@ app.directive('customPatternsInput', function () {
         $scope.aceLoaded = (editor) => {
           this.editor = editor;
           editor.getSession().setUseWrapMode(true);
+          editor.renderer.setScrollMargin(7, 7, 0, 0);
           editor.setOptions({
             highlightActiveLine: false,
             highlightGutterLine: false,
             minLines: 3,
-            maxLines: 25
+            maxLines: 25,
+            autoScrollEditorIntoView: true
```

Comment:
What problem does this solve? Might be good to add a small explainer comment.

Reply:
I was under the impression that `autoScrollEditorIntoView` was necessary to scroll to the insertion point if the user enters text while the editor is not visible in the view. Just tested it and it seems we don't need it. Removing.